```
======================================================================
ERROR: testBatch (__main__.RGBToYUVTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/lib/jenkins/workspace/tensorflow-pull-requests-cpu-python3/bazel-ci_build-cache/.cache/bazel/_bazel_jenkins/eab0d61a99b6696edb3d2aff87b585e8/execroot/org_tensorflow/bazel-out/k8-py3-opt/bin/bazel_pip/tensorflow/python/image_ops_test.runfiles/org_tensorflow/bazel_pip/tensorflow/python/ops/image_ops_test.py", line 126, in testBatch
    inp = np.random.rand(shape).astype(nptype)
  File "mtrand.pyx", line 1363, in mtrand.RandomState.rand
  File "mtrand.pyx", line 861, in mtrand.RandomState.random_sample
  File "mtrand.pyx", line 167, in mtrand.cont0_array
TypeError: 'tuple' object cannot be interpreted as an integer

----------------------------------------------------------------------
Ran 30 tests in 1.281s

FAILED (errors=1)
area_ratio_hist  [247 325 299 258 237 221 169 127  83  34]
================================================================================
```

--- a/tensorflow/python/ops/image_ops_test.py
+++ b/tensorflow/python/ops/image_ops_test.py
@@ -123,7 +123,7 @@ class RGBToYUVTest(test_util.TensorFlowTestCase):
     shape = (batch_size, 2, 7, 3)
 
     for nptype in [np.float32, np.float64]:
-      inp = np.random.rand(shape).astype(nptype)
+      inp = np.random.rand(*shape).astype(nptype)
 
       # Convert to YUV and back, as a batch and individually
       with self.test_session(use_gpu=True) as sess:

